core/src/main/java/com/github/jensborch/webhooks/WebhookConfiguration.java

```diff
@@ -0,0 +1,50 @@
+package com.github.jensborch.webhooks;
+
+import java.time.ZonedDateTime;
+import java.time.temporal.TemporalUnit;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Configuration for Webhooks4j. Users of Webhooks4j must implement a CDI
+ * producer that returns implementation of this interface.
+ * <p>
+ * Note: The TTL values are only used by the MongoDB modules, but could be used
+ * by additional persistence implementations.
+ * </p>
+ *
+ */
+public interface WebhookConfiguration {
+
+    /**
+     * Time to live (TTL) for webhook events in the events repository.
+     *
+     * @return the TTL
+     */
+    long getTimeToLive();
+
+    /**
+     * Time unit used for TTL.
+     *
+     * @return the time unit
+     */
+    TimeUnit getTimeToLiveUnit();
+
+    /**
+     * The synchronisation offset to use when synchronizing old events.
```

Comment:
The offset here is used to subtract from the last received event to get a safer starting point. I find the word "offset" a bit confusing in this context. 
I was thinking more in the line of "lookBackTime"

Reply:
I think offset can be used as it is is defined as "a amount that diminishes or balances the effect of an opposite one"